spec/requests/upload_supporting_evidence_request_spec.rb

```diff
@@ -21,67 +21,80 @@
   describe 'Post /v0/upload_supporting_evidence' do
     context 'with valid parameters' do
       it 'returns a 200 and an upload guid' do
-        post '/v0/upload_supporting_evidence',
-             params: { supporting_evidence_attachment: { file_data: pdf_file } }
+        post '/v0/upload_supporting_evidence', params: { supporting_evidence_attachment: { file_data: pdf_file } }
         expect(response).to have_http_status(:ok)
         sea = SupportingEvidenceAttachment.last
+
         expect(JSON.parse(response.body)['data']['attributes']['guid']).to eq sea.guid
         expect(sea.get_file&.read).not_to be_nil
       end
+
+      it 'creates a LighthouseSupportingEvidenceAttachment as well' do
+        post '/v0/upload_supporting_evidence', params: { supporting_evidence_attachment: { file_data: pdf_file } }
```

Comment:
Generally, `v0` has been associated with EVSS. Maybe consider creating a new controller. (Just as an example, here's what we did for Intent To File: [EVSS](https://github.com/department-of-veterans-affairs/vets-api/blob/fe39a1c2f98ca4b685d2c47e6cae6ebeba88d3ca/app/controllers/v0/intent_to_files_controller.rb#L8); [LH](https://github.com/department-of-veterans-affairs/vets-api/blob/fe39a1c2f98ca4b685d2c47e6cae6ebeba88d3ca/modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb#L9). This also makes it easier to track during the migration process, and also might help with your logging concern.